ouroboros-consensus/src/Ouroboros/Storage/ChainDB/Impl/Reopen.hs

```diff
@@ -166,3 +159,13 @@ reopen (CDBHandle varState) launchBgTasks = do
           }
 
         when launchBgTasks $ Background.launchBgTasks env replayed
+
+
+{-------------------------------------------------------------------------------
+  Auxiliary
+-------------------------------------------------------------------------------}
+
+pointToEpoch :: Monad m => EpochInfo m -> Point blk -> m EpochNo
+pointToEpoch epochInfo = \case
+    GenesisPoint      -> return 0
```

Comment:
This line seems a bit bogus, since EBB 0 demarcates the _beginning_ of epoch 0 but is also the _successor_ of `GenesisPoint`.

Harmless?

Reply:
This is just for tracing, so harmless indeed.

Reply:
Even if i _wasn't_ for tracing, it is nonetheless true that the genesis point is in epoch 0, so I think this is correct either way.